semana10/Astrotrips/src/containers/LoginPage/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { push } from "connected-react-router";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import styled from "styled-components";
+import { login } from '../../actions/login'
+
+const LoginWrapper = styled.form`
+  width: 100%;
+  height: 100vh;
+  gap: 10px;
+  place-content: center;
+  justify-items: center;
+  display: grid;
+`;
+
+class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: ""
+    };
+  }
+
+  handleFieldChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+
+  handleLoginButton = () => {
+    this.props.login(this.state.email, this.state.password)
```

Comment:
Besteirinha, mas só pra você ver um exemplo de desestruturação:

Outra forma de fazer esse método:

```js
handleLoginButton = () => {
	const { login } = this.props;
	const { email, password } = this.state;
	
	login(email, password);
}
```

Reply:
Só pra evitar a repetição do `this.blabla.bla`